SNAPSHOTS.md

```diff
@@ -0,0 +1,114 @@
+# IBM Spectrum Scale CSI driver volume snapshots
+
```

Comment:
Can we update the readme remove namespace related changes for snapshot-controller. Also we can mention this is only for k8s